nixos/modules/services/hardware/bitbox-bridge.nix

```
@@ -0,0 +1,73 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.bitbox-bridge;
+in
+{
+  options = {
+    services.bitbox-bridge = {
+      enable = lib.mkEnableOption "Bitbox bridge daemon, for use with Bitbox hardware wallets.";
+
+      package = lib.mkPackageOption pkgs "bitbox-bridge" { };
+
+      port = lib.mkOption {
+        type = lib.types.port;
+        default = 8178;
+        description = ''
+          Listening port for the bitbox-bridge.
+        '';
+      };
+
+      runOnMount = lib.mkOption {
+        type = lib.types.bool;
+        default = true;
+        example = false;
+        description = ''
+          Run bitbox-bridge.service only when hardware wallet is plugged, also registers the systemd device unit.
+          This option is enabled by default to save power, when false, bitbox-bridge service runs all the time instead.
+        '';;
+      };
```

I was typing a review comment for this, exactly when you updated :)

So as you can see using `mkEnableOption` auto appends stuff:
```
Whether to enable Run bitbox-bridge.service only.....
```

You want something like this instead:

```suggestion
      runOnMount = lib.mkEnableOption null // {
        description = ''
          Run bitbox-bridge.service only when hardware wallet is plugged, also registers the systemd device unit.
          This option is enabled by default to save power, when false, bitbox-bridge service runs all the time instead.
        '';
      };
```

You can check how it looks by downloading the artifact from our CI NixOS manal build then looking at the options.html.

Or you can build the manual yourself:

```
nix build --print-out-paths ".#htmlDocs.nixosManual.x86_64-linux"
```

---

I've changed the logic to make `config.bitbox-bridge.runOnMount = true` by default, seems like a better default to conserve power/battery. Let me know if you still think we should change it ;)

---

Please use mkEnableOption. You can still enable it by default by setting the override, just like how I did.